File: Daily Challanges/2026-04-April/2026-04-10.py
def rook_attack(rook1, rook2):
    col1, row1 = rook1
    col2, row2 = rook2
    return col1 == col2 or row1 == row2


print(rook_attack("A1", "A8"))
print(rook_attack("B4", "F4"))
print(rook_attack("E3", "D4"))

File: Daily Challanges/2026-04-April/2026-04-10.js
function rookAttack(rook1, rook2) {
  const [col1, row1] = rook1
  const [col2, row2] = rook2
  return col1 == col2 || row1 == row2;
}

console.log(rookAttack("A1", "A8"))
console.log(rookAttack("B4", "F4"))
console.log(rookAttack("E3", "D4"))